packages/ag-charts-enterprise/src/features/zoom/zoom.test.ts

```diff
@@ -463,8 +463,7 @@ describe('Zoom', () => {
             };
             await prepareChart(undefined, undefined, options);
             await scrollAction(cx, cy, -1)(chart);
-            await chart.update({
-                ...options,
+            await chart.updateDelta({
```

Comment:
This update wasn't applying the same options, as `options` (despite appearing to do so), as it didn't include a bunch of mutations in `prepareChart()` - switching to `updateDelta()` achieves the intended outcome.